docuilib/src/components/pageComponents/DefaultTabs.ts

```diff
@@ -0,0 +1,32 @@
+export const devTab = {
+  title: 'Dev',
+  sections: [
+    {
+      title: 'Usage',
+      type: 'usage'
+    },
+    {
+      type: 'info'
+    },
+    {
+      type: 'tip'
+    },
+    // {
```

Comment:
You can remove the commented code here, this section can be added in noci

Reply:
Fixed